api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -26,24 +13,49 @@
 	
 	<bean id="openconceptlab.importService"
 		class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
-		<property name="transactionManager">
-			<ref bean="transactionManager" />
-		</property>
+		<property name="transactionManager" ref="transactionManager"/>
 		<property name="target">
 			<bean class="org.openmrs.module.openconceptlab.ImportServiceImpl">
 				<property name="sessionFactory" ref="dbSessionFactory" />
 				<property name="adminService" ref="adminService" />
 				<property name="conceptService" ref="conceptService"/>
+				<property name="oclConceptService" ref="openconceptlab.conceptService" />
 			</bean>
 		</property>
-		<property name="preInterceptors">
-			<ref bean="serviceInterceptors" />
+		<property name="preInterceptors" ref="serviceInterceptors"/>
+		<property name="transactionAttributeSource" ref="transactionAttributeSource"/>
+	</bean>
+
+	<bean parent="serviceContext">
+		<property name="moduleService">
+			<list>
+				<value>org.openmrs.module.openconceptlab.ImportService</value>
+				<ref bean="openconceptlab.importService"/>
+			</list>
 		</property>
-		<property name="transactionAttributeSource">
-			<ref bean="transactionAttributeSource" />
+	</bean>
+
+	<bean id="openconceptlab.conceptService"
+		  class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
+		<property name="transactionManager" ref="transactionManager"/>
+		<property name="target">
+			<bean class="org.openmrs.module.openconceptlab.OclConceptServiceImpl">
+				<property name="dbSessionFactory" ref="dbSessionFactory" />
+			</bean>
 		</property>
+		<property name="preInterceptors" ref="serviceInterceptors"/>
+		<property name="transactionAttributeSource" ref="transactionAttributeSource"/>
 	</bean>
-	
+
+	<bean parent="serviceContext">
+		<property name="moduleService">
+			<list>
+				<value>org.openmrs.module.openconceptlab.OclConceptService</value>
+				<ref bean="openconceptlab.conceptService"/>
+			</list>
+		</property>
+	</bean>
```

Comment:
Do we need to list this twice, or can we list it once and have the two services in the list?

Reply:
This is probably something we should fix at some point, but [the `setModuleServiceMethod()`](https://github.com/openmrs/openmrs-core/blob/011d738ee5f1a325d38c2cac588fe7446de7885a/api/src/main/java/org/openmrs/api/context/ServiceContext.java#L739) only processes the first two arguments, so it's necessary to invoke this twice.